Assets/MixedRealityToolkit/Interfaces/SceneSystem/IMixedRealitySceneSystem.cs

```diff
@@ -0,0 +1,245 @@
+        /// If a scene is not loaded, it will be ignored.
+        /// If a scene operation is in progress, no action will be taken.
+        /// </summary>
+        /// <param name="scenesToLoad"></param>
+        /// <returns>Task</returns>
+        Task UnloadContent(IEnumerable<string> scenesToUnload);
+
+        /// <summary>
+        /// Async method to load a single scene by name.
+        /// If a scene operation is in progress, no action will be taken.
+        /// </summary>
+        /// <param name="sceneToLoad">Name of content scene to load. Invalid scenes will be ignored.</param>
+        /// <param name="mode">Additive mode will load the content additively. Single mode will first unload all loaded content scenes first.</param>
+        /// <param name="activationToken">
+        /// Optional token for manual scene activation. Useful for loading screens and multiplayer.
+        /// If not null, operation will wait until activationToken's AllowSceneActivation value is true before activating scene objects.
+        /// </param>
+        /// <returns>Task</returns>
+        Task LoadContent(string sceneToLoad, LoadSceneMode mode = LoadSceneMode.Additive, SceneActivationToken activationToken = null);
+
+        /// <summary>
+        /// Async method to unload a single scene by name.
+        /// If the scene is not loaded, no action will be taken.
+        /// If a scene operation is in progress, no action will be taken.
+        /// </summary>
+        /// <param name="sceneToUnload"></param>
+        /// <returns>Task</returns>
+        Task UnloadContent(string sceneToUnload);
+
+        /// <summary>
+        /// Async method to load content scenes by tag. All scenes with the supplied tag will be loaded.
+        /// If no scenes with this tag are found, no action will be taken.
+        /// If a scene operation is in progress, no action will be taken.
+        /// </summary>
+        /// <param name="tag">Scene tag.</param>
+        /// <param name="mode">Additive mode will load the content additively. Single mode will first unload all loaded content scenes first.</param>
+        /// <param name="activationToken">
+        /// Optional token for manual scene activation. Useful for loading screens and multiplayer.
+        /// If not null, operation will wait until activationToken's AllowSceneActivation value is true before activating scene objects.
+        /// </param>
+        /// <returns>Task</returns>
+        Task LoadContentByTag(string tag, LoadSceneMode mode = LoadSceneMode.Additive, SceneActivationToken activationToken = null);
+
+        /// <summary>
+        /// Async method to unload scenes by name.
+        /// If a scene is not loaded, it will be ignored.
+        /// If a scene operation is in progress, no action will be taken.
+        /// </summary>
+        /// <param name="tag">Scene tag</param>
+        /// <returns>Task</returns>
+        Task UnloadContentByTag(string tag);
```

Comment:
If the scene is not tagged, is it unable to be unloaded?

Reply:
Correct, at least not by this method. You can still use:
```
Task UnloadContent(string sceneToUnload);
Task UnloadContent(IEnumerable<string> scenesToUnload);
```